WORKSPACE.bazel

```diff
@@ -81,8 +84,20 @@ llvm_toolchain(
     # https://github.com/grailbio/bazel-toolchain/blob/ceeedcc4464322e05fe5b8df3749cc02273ee083/toolchain/cc_toolchain_config.bzl#L148-L150
     link_libs = {
         "": ["--driver-mode=g++"],
+        "linux-x86_64": ["--driver-mode=g++"] + [
+            opt.format(sysroot = SYSROOT_PATH)
```

Comment:
hmm, surprised setting sysroot below isn't enough. Is this a bug in the llvm toolchain?

Reply:
Not sure. It's what I do here:
https://github.com/oliverlee/bazel_bootlin/blob/81204765683aa144fe537da97b0a1d6c3e4eac38/toolchain/cc_toolchain_config.bzl#L70-L76

And maybe some other toolchains do the same:
https://github.com/aspect-build/gcc-toolchain/pull/6/files